python/ray/dataframe/test/test_dataframe.py

```diff
@@ -1247,20 +1251,20 @@ def test_fillna_inplace(num_partitions=2):
     df[1][:4] = np.nan
     df[3][-4:] = np.nan
 
-    ray_df = rdf.from_pandas(df, num_partitions)
+    ray_df = from_pandas(df, num_partitions)
     df.fillna(value=0, inplace=True)
     assert not ray_df_equals_pandas(ray_df, df)
 
     ray_df.fillna(value=0, inplace=True)
     assert ray_df_equals_pandas(ray_df, df)
 
-    ray_df = rdf.from_pandas(df, num_partitions).fillna(value={0: 0},
+    ray_df = from_pandas(df, num_partitions).fillna(value={0: 0},
```

Comment:
the indentation of the next line looks a bit off

Reply:
Fixed!